src/components/auth/useTorus.js

```diff
@@ -17,14 +17,117 @@ export const torus = new TorusSdk({
   ...sdkOptions,
 })
 
+class TorusLogin {
```

Comment:
Better to move it to the separate module (like we moved ZoomSDK class separately from the hooks)

Reply:
its all the same logic and its not that big. so for readability i'd keep it in same file